core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -414,6 +414,9 @@ $(document).ready(function () {
                         },
                         {
                             data: function (row, type) {
+                                if (typeof row.peakMemoryMetrics == "undefined") {
```

Comment:
It's better to use `===` and `!==` rather than `==` and `!=`.

Reply:
Fixed